data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+				launch
+	
+	on enter
+		dialog "As your repulsor engines roar, you notice a large pirate ship looming over you from space. Even from the underside, it is undeniably a Bactrian, but as to how a pirate could ever got ahold of one is a mystery."
+		dialog `	You receive a transmission from Lieutenant Paris. "I don't know how they have a Bactrian, but I suggest we jump out of here immediately and make it back to the Deep alive."`
+	
+	npc save escort accompany
+		personality timid escort
+		government Republic
+		ship "Star Queen" "D.S.S. Alcubierre"
+	
+	on visit
+		dialog `You have landed on <planet>, but you left the <npc> behind! Wait for it to enter the system before landing.`
+	
+	on complete
+		payment 100000
+
+mission "Bactrian Unlock [2]: Pirate Bactrian"
+	landing
+	invisible
+	destination Valhalla
+	to offer
+		has "Bactrian Unlock [1]: done"
+	npc kill
+		government Pirate
+		personality staying nemesis waiting
+		ship "Bactrian" "Cosmic Devil"
+		dialog `The Bactrian "<npc>" has been destroyed. Now the pirates here will have a harder time attacking the Republic.`
+
+mission "Bactrian Unlock [2]: More Pirates"
+	landing
+	invisible
+	to offer
+		has "Bactrian Unlock [1]: done"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		government Pirate
+		personality staying nemesis waiting
+		ship "Leviathan (Laser)" "Eve"
+		ship "Leviathan (Laser)" "Cain"
+
+mission "Bactrian Unlock [3]"
+	name "Take down the Devil"
+	description "You have returned the scientists, but the pirates still have a Bactrian. Eliminate it and return to <destination>."
+	landing
+	destination Valhalla
+	waypoint Arneb
+	to offer
+		has "Bactrian Unlock [2]: done"
+		not "Bactrian Unlock: Pirate Bactrian: done"
+		not "Bactrian Unlock [3-alt]: offered"
+	on offer
+		conversation
+			`As you land near the Star Queen, you can see the passengers of the ship being escorted off of it and individually checked to make sure everyone is fine. Sadly, you also notice a number of body bags next to the ship, likely from scientists or crew that were found dead on the ship.`
+			`	Lieutenant Paris is waiting for you outside of your ship, and hands you another 100,000 credits when you walk up to him.`
+			`	"Thanks for all the help, Captain <last>. We couldn't have done it without you."`
+			`	Paris looks back to the Star Queen, and you can overhear talk of who survived and who died. "Sarah, Nicole, Abby..." the list of names goes on for about two dozen more, most of which sound female. "...and Benjamin are all either missing or were found dead on board. The surviving crew tells us that some were killed when they were captured or thrown out the airlock for struggling, but most of the others were either killed or dragged off at the facility."`
```

Comment:
I know why I thought they were dead. Because their names immediately followed a sentence that ended "you can overhear talk of who survived and who died." It's only afterwards that you read that all of those mostly women are dead _or missing._